docs/concepts/feature-registry.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Feature Registry
+parent: Feathr Concepts
+---
+
+# Feature Registry and Feathr UI
+
+Feature registry is an important component of a feature store. This documentation will cover the supported backend of a feature registry and the usages.
+
+## Introduction
```

Comment:
Can we add some content under this section:

You can use Feathr UI to 
- Identity Data Sources for features
- Search features and explore feature metadata details
- Track project and feature level lineage data
- Manage access control